### PR TITLE
Replace json-styled translations with string-value translations

### PR DESCRIPTION
**Pull request by HenryLoenwind**
Replaced the json-styled translations that were used by the configuration screen with pure text (vanilla format) versions.

Adapted the configuration screen to apply the formatting in code.

A number of keys had to be split for this to work.

### CODE REVIEW

**Comment by sciwhiz12** [comment]
To add context, after discussion in Discord prompted by me, we decided that it would be best for NeoForge's language files to only use string-values in its translations.

This is primarily because Crowdin does not understand the extended syntax that some translation keys currently use, leading to problems such as Crowdin mistakenly allowing translators to translate the `color` field of components in the extended syntax.

Mods are free to use the extended syntax (as it stands now or whatever it may be in the future) based on their own design decisions on how they will handle translations, but as far as NeoForge itself is concerned, our use of Crowdin necessarily locks us into using string values. (Unless we find a way to get Crowdin to understand the extended syntax, but that's unlikely in my opinion.)

File: src/main/resources/assets/neoforge/lang/en_us.json [hunk 1]
```diff
@@ -147,51 +147,15 @@
...
-      "text": " > ",
-      "color": "gold",
-      "bold": true
-    },
-    {
-      "index": 1
-    }
-  ],
+  "neoforge.configuration.uitext.breadcrumb.order": "%1$s%2$s%3$s",
+  "neoforge.configuration.uitext.breadcrumb.seperator": " > ",
```
Comment (sciwhiz12) [inline src/main/resources/assets/neoforge/lang/en_us.json]: Should we perhaps combine these two into a single translation key? Having the two be separated doesn't sit right with me, and seems a bit too heavy a burden to understand for a translator.

It should be possible to combine them, and still have the separator be colored. Based on my testing, you'd have to apply the gold and boldface formatting on the combined breadcrumb translation, and explicitly set no-boldface and white formatting on the individual titles. (When composing the breadcrumb translation, only the 'child' or second parameter is set to explicit no-boldface and white, assuming the parent is already set as such.)

Here's some example code of what I mean:
```java
Style titleStyle = Style.EMPTY.withColor(CommonColors.WHITE).withBold(false);
Style crumbStyle = Style.EMPTY.withColor(ChatFormatting.GOLD).withBold(true);
String crumb = "%1$s > %2$s";
//String crumb = "%2$s < %1$s";

List<String> trail = List.of("Apple", "Banana", "Cherry", "Durian");

MutableComponent component = Component.literal(trail.getFirst()).withStyle(titleStyle);
for (String s : trail.subList(1, trail.size())) {
    component = Component.translatable(crumb,
            component,
            Component.literal(s).withStyle(titleStyle)
    ).withStyle(crumbStyle);
}
```

If that isn't acceptable, then at least move the spaces from the `.separator` to the `.order` to visibly separate the three positional parameters.

**Review by sciwhiz12 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/neoforge/lang/en_us.json (hunk 1 above)
Comment (HenryLoenwind) [reply src/main/resources/assets/neoforge/lang/en_us.json]: I'm not a fan of forcing a hardcoded copy of the default onto elements. Such stuff tends to come apart at the seams with time...

File: src/main/resources/assets/neoforge/lang/en_us.json [hunk 2]
```diff
@@ -154,8 +154,8 @@
   "neoforge.configuration.uitext.longstring": "This value is too long to be edited in the UI. Please edit it in the config file.",
   "neoforge.configuration.uitext.section": "%s...",
   "neoforge.configuration.uitext.sectiontext": "Edit",
-  "neoforge.configuration.uitext.breadcrumb.order": "%1$s%2$s%3$s",
-  "neoforge.configuration.uitext.breadcrumb.seperator": " > ",
+  "neoforge.configuration.uitext.breadcrumb.order": "%1$s %2$s %3$s",
+  "neoforge.configuration.uitext.breadcrumb.seperator": ">",
```
Comment (sciwhiz12) [inline src/main/resources/assets/neoforge/lang/en_us.json]: I just realized: the proper spelling is "sep**a**rator", not "sep**e**rator". (Note the second vowel.) I think it's a minor enough change that we can include that in this PR.

**Comment by sciwhiz12** [comment]
Other than my remaining suggested change, the PR looks fine to me. Once it's done, I'll merge it in short order.

File: src/main/resources/assets/neoforge/lang/en_us.json (hunk 2 above)
Comment (HenryLoenwind) [reply src/main/resources/assets/neoforge/lang/en_us.json]: It's a new key anyway.

And really...English needs to start spelling its əs as ə, not as random other letters.

**Comment by qznfbnj** [comment]
Is this intended?
image

**Comment by sciwhiz12** [comment]
> Is this intended? _-- snipped image --_

@qznfbnj, what are you referring to?

**Comment by qznfbnj** [comment]
> > Is this intended? _-- snipped image --_
> 
> @qznfbnj, what are you referring to?

image

**Comment by sciwhiz12** [comment]
Ah, that makes it more clear. @HenryLoenwind, could you please fix that minor issue as well?

**Review by sciwhiz12 — APPROVED** [review]

**Pull request merged**